### ISSUE TITLE
Visual Bug on docs website 

### ISSUE THREAD

**Issue by nivekithan** [remix-run/remix#3789]
### What version of Remix are you using?

Does not matter

### Steps to Reproduce

Go to https://remix.run/docs/en/v1/api/remix#unstable_parsemultipartformdata

You can see `unstable_parseMultipartFormData` getting `overflow`

> NOTE: My viewport width is 1536

![image](https://user-images.githubusercontent.com/52579256/179572735-9d220c6c-6843-44d4-a7ee-bb3f9b226353.png)

### Expected Behavior

Not getting overflow

### Actual Behavior

Getting overflow

![image](https://user-images.githubusercontent.com/52579256/179572924-c168cf53-d555-4a56-9ea1-0273326acc5b.png)

**Comment by Abhiram-Gullapalli** [comment]
Was the issue closed, because I didn't find any such glitches on my device. What was the resolution of the screen you viewed it on?

**Comment by nivekithan** [comment]
@Abhiram-Gullapalli  yep I am still facing that issue 
![image](https://user-images.githubusercontent.com/52579256/183129039-67ed8206-80f9-4397-b1a1-2b2103381b4c.png)

my display resolution is `1920 x 1080`

**Comment by Abhiram-Gullapalli** [comment]
I still can't observe similar behaviour, despite inspecting with the mentioned screen resolution!

**Comment by nivekithan** [comment]
I am using windows, on brave browser with version   
```
Version 1.42.86  Chromium: 104.0.5112.81 (Official Build)  (64-bit)
```

My viewport width was ` 1536`

**Comment by vincecao** [comment]
Add below style in `h2` should resolve the issue

``` css
overflow: hidden;
text-overflow: ellipsis;
white-space: nowrap;
```

Or if using tailwindcss, could just add `truncate`. 

<img width="1416" alt="Screen Shot 2022-08-06 at 11 30 22 PM" src="https://user-images.githubusercontent.com/17363908/183278325-ae084098-f4d0-4f69-8a5e-ef772ba6ab66.png">

Unfortunately, I did not find where style of api doc is generated for now. If I found later, I might can create fix PR.
